task5/src/main/java/PageServlet.java

```diff
@@ -0,0 +1,25 @@
+import javax.servlet.*;
+import javax.servlet.http.*;
+import javax.servlet.annotation.*;
+import java.io.*;
+
+@WebServlet(name = "PageServlet", value = "/page")
+public class PageServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+
+        //RequestDispatcher view = request.getRequestDispatcher("/WEB-INF/page.jsp");
+        //view.forward(request, response);
+        InputStream is = getServletContext().getResourceAsStream("/WEB-INF/page.html");
```

Comment:
If use sendRedirect, we get url =".../page.html", but need /page
When I used forward to the html file returns empty page, with jsp worked correctly. I didnt find solution for the problem for html

Reply:
... Then forward, instead of redirect)
Anyway.,, when the user types /page, and sees the content of page.html, hardly anyone can say this is not accordion to the task) What the browser displays in address barm is some other question) But I do accept your solution